src/main/java/profile_manager/frameworks_and_drivers/ProfileManagerDataAccess.java

```diff
@@ -0,0 +1,134 @@
+package profile_manager.frameworks_and_drivers;
+
+import java.io.*;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.*;
+import profile_manager.application_business_rules.ProfileRepoInt;
+import profile_manager.application_business_rules.ProfileRepoRequestModel;
+
+public class ProfileManagerDataAccess implements ProfileRepoInt {
+    private final File csvFile;
+
+    private final Map<String, Integer> headers = new LinkedHashMap<>();
+
+    private final Map<String, ProfileRepoRequestModel> accounts = new HashMap<>();
+    public ProfileManagerDataAccess(String csvFilePath) throws IOException {
+        csvFile = new File(csvFilePath);
+
+        headers.put("userName", 0);
+        headers.put("profileName", 1);
+        headers.put("dob", 2);
+        headers.put("description", 3);
+        headers.put("socialLinks", 4);
+        headers.put("sensitiveWords", 5);
+        headers.put("interests", 6);
+        headers.put("groups", 7);
+        headers.put("friends", 8);
+        headers.put("blockedUsers", 9);
+        headers.put("creationTime", 10);
+
+        if (csvFile.length() == 0) {
+            save();
+        } else {
+            BufferedReader reader = new BufferedReader(new FileReader(csvFile));
+            reader.readLine(); // skip header
+
+            String row;
+            while ((row = reader.readLine()) != null) {
+                String[] col = row.split(",");
+                String userName = String.valueOf(col[headers.get("userName")]);
+
+                String profileName = String.valueOf(col[headers.get("profileName")]);
+                LocalDate dob = LocalDate.parse(col[headers.get("dob")]);
+                String description = String.valueOf(col[headers.get("description")]);
+
+                String socialLinks = String.valueOf(col[headers.get("socialLinks")]);
+                List<String> socialLinksList = Arrays.asList(socialLinks.split(";"));
+
```

Comment:
For each of these that involve a list, you'll need to add a condition for if the cell is empty. For example, if the profile has nothing listed for social links, String socialLinks = String.valueOf(col[headers.get("socialLinks")]) will return "" (an empty string). Therefore when you get to the next line, the socialLinks list looks like [""] instead of []. 

I'm not sure if this will cause functional errors but I noticed it as an issue for the formatting of the csv file.